### PR TITLE
REVERT #6072, cause a pr was approved in less than 7 hours and without even TM...

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
reverts the #6072

## Why It's Good For The Game
survivors should be a viable role. and killing it cause benos LOST for survivors just serve as proof of robust players or weak players depending on which side you are. a couple survivors winning over a hive.. will only be a shorter match than a FUCKING ARMY stomping said hive. ((if the hive cant fight off.. a couple of dudes with improvised weapons. imagine a whole army))

## Changelog
:cl:
tweak: gets survivors back in town
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -39. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by KruatKutter** [comment]
Xenos Mad

**Comment by BeloneX** [comment]
Circlejerk revert.

**Comment by Swagile** [comment]
but why was survivor removed in the first place

good survivors:

A) have to rush good guns
B) have to rush for meds if they want to survive more than 1 minute
C) [optional] get the goodies specific to the map, like special guns or engineering materials caches like in LV domes

all of this takes about 5-10 minutes depending on survivor spawn and how fast said player can pull it off

xenos are mature or even elder by then depending on caste, and with a queen or even shrike that knows their shit, they can easily stomp a couple of survivors

if the issue is survivors spawning then immediately rushing young xenos, one screech or shrike punt then surround kills that off. if its THAT THAT bad, remove all survivor spawns near caves, because most survivor spawns have shit weapons anyway; only 1 survivor spawn has a shotgun last i recall, the rest have a mosin, lever action, shitty Mp5k, and one has a FAMAS / Pulse rifle

tl;dr if you die to survivors your fucking bad, and removing survivors because of that is dumb

**Comment by Sulaboy** [comment]
Survivor shouldn’t be removed pending some tweaking. It’s decent enough to provide Xenos something to do during the prep phase besides slashing lights.

**Comment by Texicos** [comment]
Bring back the funny barman

**Comment by UntoldTactics** [comment]
I don't see any point to keep survivors.
Beyond marine mains finding them fun and the arguments of 'xenos should just be robust lol'

1) What is the point to survivors, what do they contribute to the round, and in extremes, do they affect balance?
2) The majority of those petitioning for survival re-add are marine mains, and frankly survivor is a useless role that brings no benefit to them, or to anyone else, all they serve is to annoy xenos and occasionally kill them.

Give me any good reasons why they should exist beyond 'Why not, xenos should just robust them', that's all we have heard and nothing else, no good reasons to keep them nor any reasons why they shouldn't be kept beyond illogical opinions based on skill. 

All I have seen and heard is marines going 'if you die to them you just suck', no reason as to WHY they should be kept, what they contribute to the game, and if the reward of keeping them but annoying the fuck out of the xeno mains is actually worth having them in the first place.

Just because you find it fun doesn't mean others do.

**Comment by Sulaboy** [comment]
> Give me any good reasons why they should exist beyond 'Why not, xenos should 
> 
It gives Xenos something to do for the first 30 minutes besides slashing lights.

**Comment by fernandos13** [comment]
`Give me any good reasons why they should exist beyond 'Why not, xenos should`

i mean we play the game cause its fun
playing survivor is intense as shit. and each fight is very hard to tackles as a survivor.

one might say the "dont bring anything in the game for marines"
for which i say. well why should we have the ERT then? or the XRT? or why should we have any role aside marine. hell why have MR and CL.. or even RO... why have a CSE. why have a dedicated engineer.. 

its another HUMAN role to play. you dont need to bring stuff to marines. its not about that
if you fight with benos its already enough for you (and benos) to have fun.. after all whats TGMC about? isn't it about xenos killing humans and humans killing xenos or the fight that incurs... its gameplay for both. sides. and marines are a rather different enemie than marines since they are more jagged. and rough usually but also more. rushy...its diferent and getting back to "they dont bring anything for marines" well they do have rescues... they do leave.. signs of strugge that is a extra salt on the static map. they do modify the terrain building and moving improvised barricades and welded vents..  or just extra larvas and bodies...

as for benos

well you rather just.. sit there... and wait for marines to come? why not start the reason you joined the game "fighting" early. if they are SO EASY to stomp... then why dont benos just stomp them.. and if they are "SO HARD" to deal with. not dont they just win all the times.. 

sure SOMETIMES you get a survivor major. and everyone gets impressed for it cause.... its.. not that easy to achieve...

**Comment by revelation-8468** [comment]
I remember seeing someone cry about survs killing AFK xenos. Removing surv and forcing most xenos to essentially waste 30 minutes of their time does not fix this.

If anything, it makes it worse. I can still droppod in at 12:28 and theoretically kill several AFK xenos and get an early Marine Major. Or, even - kill the pools early because the Hivelord is AFK watching youtube, waiting for the round to ACTUALLY start at 12:33.

If you think hunting down survivors (which can be killed easily with stuns) is less fun then clicking on APCs and lights for half an hour then there may be something off.

**Comment by TiviPlus** [comment]
image
I was very clear on this topic I believe
